### PR TITLE
add customizations to graylog chart

### PR DESCRIPTION
**Pull request by chrkaatz**
* master service port and annotations
* env vars for the pods through the stateful set

Signed-off-by: Christian Kaatz <chrkaatz@googlemail.com>

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by chrkaatz** [comment]
This is the successor of #12348.

File: stable/graylog/README.md [hunk 1]
```diff
@@ -96,8 +96,11 @@ The following table lists the configurable parameters of the Cassandra chart and
 | `graylog.affinity`                      | Graylog server affinity                                                                                                                               | `{}`                                  |
 | `graylog.tolerations`                   | Graylog server tolerations                                                                                                                            | `[]`                                  |
 | `graylog.nodeSelector`                  | Graylog server node selector                                                                                                                          | `{}`                                  |
+| `graylog.env         `                  | Graylog server env variables                                                                                                                          | `{}`                                  |
```
Comment (KongZ) [inline stable/graylog/README.md]: Just small change, remove redundant space

Comment (chrkaatz) [reply stable/graylog/README.md]: done

**Comment by rakeshpatri** [comment]
There is an issue with the serviceType NodePort for this release. 
 In case of service type "NodePort", both the master service and web service picking the same nodePort settings, hence one of the services doesn't come up. We need to allow users the specify a separate nodePort for master and web.

**Pull request merged**